spec/models/user_spec.rb

```diff
@@ -1,5 +1,93 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
```

Comment:
very thorough tests! some of these for checking valid attributes could be done with shoulda:
```
it { should have_valid(:screen_name).when("tester") }
it { should_not have_valid(:screen_name).when(nil, "") }
```

Reply:
We'll look into "shoulda" tests to familiarize ourselves with the syntax, and will look into use shoulda tests going forward!